src/components/ProfilePic/SideBarProfile.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { makeStyles, Avatar, Typography } from "@material-ui/core";
+import { styles } from "../../styles/styles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+    minWidth: styles.sizes.sidebarDimensions.minWidth,
+    maxWidth: styles.sizes.sidebarDimensions.maxWidth,
+    backgroundColor: styles.colors.darkGray,
+  },
+  pic: {
+    width: "100%",
+    height: "100%",
+    minHeight: styles.sizes.sidebarDimensions.minWidth,
+    maxHeight: "auto",
+  },
+  text: {
+    color: "white",
+    backgroundColor: styles.colors.transparentLightBlue,
+    padding: styles.sizes.specialThick.spacingThick1,
+    position: "relative",
+    bottom: styles.sizes.thickness.thick22
+  }
+}));
+
+const ProfilePic = (props) => {
```

Comment:
Please destructure the props and use the ones neded, never receive props without destructuring unless necessary